components/Button.tsx

```diff
@@ -3,21 +3,32 @@ import { FC, PropsWithChildren } from 'react';
 interface Props {
   form?: string;
   disabled: boolean;
+  variant: string;
   onClick?: () => void;
 }
 
 const Button: FC<PropsWithChildren<Props>> = ({
   form,
   children,
   disabled,
+  variant,
   onClick,
 }) => {
+  let classStyling: string = '';
+  if (variant == 'inputForm') {
```

Comment:
Prefer dashes instead of camel case for variant names, like CSS property names. Do this for my other suggestions too.
```suggestion
  if (variant == 'input-form') {
```